frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/state.ts

```diff
@@ -16,7 +17,7 @@ export const initialState: State = {
   showNodesListOnLVS: false,
   diskType: 'All',
   diskMode: diskModeDropdownItems.BLOCK,
-  deviceType: [],
+  deviceType: [deviceTypeDropdownItems.DISK, deviceTypeDropdownItems.PART],
```

Comment:
Use this object instead.
https://github.com/openshift/console/blob/ef18881cf0cf1b640a631c9ddc6f2f979309ca67/frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/types.ts#L12-L15